handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -89,6 +94,24 @@
             destroyedUpdater = AtomicIntegerFieldUpdater.newUpdater(OpenSslEngine.class, "destroyed");
         }
         DESTROYED_UPDATER = destroyedUpdater;
+
+        Method getUseCipherSuitesOrderMethod = null;
+        Method setUseCipherSuitesOrderMethod = null;
+        if (PlatformDependent.javaVersion() >= 8) {
+            try {
+                getUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("getUseCipherSuitesOrder");
+                SSLParameters parameters = new SSLParameters();
+                Boolean order = (Boolean) getUseCipherSuitesOrderMethod.invoke(parameters);
```

Comment:
is the rational for not using `boolean` because you don't want to risk unboxing? didn't we do some investigation into this recently .... i thought the jvm was smart enough to keep the primitive types so there would be no need to go to `Boolean` ... @buchgr wdyt?